libbeat/reader/multiline/multiline.go

```diff
@@ -326,6 +332,13 @@ func (mlr *Reader) addLine(m reader.Message) {
 
 	}
 
+	if len(m.Original) > 0 {
+		if len(mlr.message.Original) > 0 {
+			mlr.message.Original = append(mlr.message.Original, mlr.separator...)
+		}
+		mlr.message.Original = append(mlr.message.Original, m.Original...)
+	}
```

Comment:
Is there an upper limit on the size of `mlr.message.Original`? When encoding to JSON, these might become quite big. At worst events might be dropped due to the increased size.

The multiline reader already ignores new lines once `max_bytes` have been reached. So to keep memory usage at least somewhat under control.

Reply:
In case we introduce a `max_bytes`, will me make the content of `log.original` valid json or would it miss the last part?

Reply:
it would miss the last part, as is the case for unparsed raw text.

The default limit for message is already 10MB. We have users putting it to values like 250MB (xml). Do we really want to deal with potential 500MB events or bigger. I understand the sentiment of really keeping the original, but there is a limit of what can be shipped and what is accepted by the sinks.

Reply:
I'm ok with dropping as long as we have a config option which the user can adjust. Also we should at least on the debug level log when we drop things.